utils/tests/utils.go

```diff
@@ -706,6 +703,17 @@ func RedirectLogOutputToNil() (previousLog log.Log) {
 	return previousLog
 }
 
+// Set progressbar.ShouldInitProgressBar func to always return true
+// so progress bar library will be initialized and progress will be displayed.
+// The return callback set back the original func.
```

Comment:
return --> returned
The return callback set back the original func. --> The returned callback sets the original func back.